### ISSUE TITLE
How to reschedule a task ?

### ISSUE THREAD

**Issue by vaimdev** [arteria/django-background-tasks#107]
If I have run a task with schedule 

notify_user(user.id, schedule=timedelta(minutes=20)) # 20 minutes from now

And later I realise I shoud run it at another time , how can I do it ?

Do we have a task id that returned that we can edit the schedule time  ?

Besides, I have a task to be run with different parameters value , can they be run as a different tasks ?

For example:
notify_user(user.id, schedule=timedelta(minutes=20)) # 20 minutes from now

notify_user(user.id, schedule=timedelta(minutes=60)) # 60 minutes from now

first notify_user & 2nd notify_user are different user tasks.

Basically , I need to provide a UI for user to schedule tasks , how can I allow change of schedule time if existing registered task ? Can we somehow unregister it and register again ?

**Comment by kckaiwei** [comment]
Well, tasks are given a hash, under task_hash, as well as an ID in the database under the table called 'background_task'. 

If you modify the entry in the database, you can change the run time, as well as task parameters, such as, repeat intervals, repeat_until, etc.

Unregistering would be deleting that task before it is processed. Registering would be calling that method again with the proper parameters.

**Comment by vaimdev** [comment]
Hi, thanks for your explanation.
